docker/powermon.yaml

```diff
@@ -16,7 +16,7 @@ commands:
     outputs:
       - type: mqtt
         format: 
-        - type: mqttmessage
+          type: mqttmessage
```

Comment:
this change is to match current config_model, question is should we allow multiple formats (in which case we'd want a list with a `-`) and need to change some logic